agent/build.gradle

```diff
@@ -68,7 +68,11 @@ task relocateShadowJar(type: ConfigureShadowRelocation) {
 tasks.shadowJar.dependsOn tasks.relocateShadowJar
 
 task asyncProfilerLib {
-    def suffixes = ['linux-aarch64', 'linux-arm', 'linux-x64', 'linux-x86', 'macos-x64']
+    def suffixes = [
+        ['linux-arm64', 'tar.gz'],
+        ['linux-x64', 'tar.gz'],
+        ['macos', 'zip']
+    ]
```

Comment:
async-profiler author drops linux-x86 and linux-arm binary from releases since async-profiler 2.5.
https://github.com/jvm-profiling-tools/async-profiler/discussions/578

Reply:
async-profiler author provides macos archive as zip since 2.5